Client.java

```diff
@@ -0,0 +1,39 @@
+package lesson2_4.online;
+
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.image.Image;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+public class Client extends Application {
+
+
+
+    public static void main(String[] args) {
+
+        launch(args);
+    }
+
+    private static Scene scene;
+
+    @Override
+    public void start(Stage primaryStage) throws Exception {
+        scene = new Scene(loadFxml("chat.fxml"), 400, 600);
+        primaryStage.getIcons().add(new Image(getClass().getResourceAsStream("g512.png")));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [UI: Usage of GetResource may be unsafe if class is extended (UI_INHERITANCE_UNSAFE_GETRESOURCE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#ui-usage-of-getresource-may-be-unsafe-if-class-is-extended-ui-inheritance-unsafe-getresource).
Calling this.getClass().getResource(...) could give results other than expected if this class is extended by a class in another package.